tests/grammar_build_test.py

```diff
@@ -0,0 +1,26 @@
+import os
+import hashlib
+
+
+def test_ensure_grammar_file_is_built():
+    tests_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '.'))
+    grammar_file = open(f'{tests_dir}/../LATEX.g4', 'r')
+    h = hashlib.new('sha256')
+    h.update(grammar_file.read().encode('utf-8'))
+    grammar_file.close()
+    file_hash = h.hexdigest()
+    assert file_hash == '8532f17fe8000457059e7d1f8f7da18b73fffecf8a5b7830dab0bbb7d9f199b8', \
+        'LATEX.g4 has changed. Please run the build.sh script then update the hash' + \
```

Comment:
@jkdufair i think you mean `compile.sh`? also might be good to say to run it for all archs

Reply:
@builtbywill Oooh thanks. Updated.

Reply:
@jkdufair nice!